tesla_powerwall/api.py

```diff
@@ -166,97 +167,106 @@ def login(
             },
         )
 
-    def logout(self) -> None:
+    async def logout(self) -> None:
         if not self.is_authenticated():
             raise ApiError("Must be logged in to log out")
         # The api unsets the auth cookie and the token is invalidated
-        self.get("logout")
+        await self.get("logout")
+
+    async def close(self):
```

Comment:
```suggestion
    async def close(self) -> None:
```

mypy complained this was untyped